src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -7,32 +7,74 @@
 
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
 
 @NoArgsConstructor
 @AllArgsConstructor
-@Getter
-@Setter
```

Comment:
moim zdaniem adnotacja @Getter i @Setter jest o wiele bardziej czytelna niż pisanie 'z ręki' metod - nie jest to błąd ale kod staje się mniej czysty

Reply:
Mysle ze tutaj ustalimy na spotkaniu wspolna wersje .